src/commands/remove/remove_package.py

```diff
@@ -6,6 +6,7 @@
 from src.commands.remove import removal_exceptions
 
 
+# TODO: https://github.com/software-mansion/protostar/issues/241
```

Comment:
We do we need this comment? We captured issue already

Reply:
I bet we will close the issue and it will remain :p

Reply:
This command should be in the `remove_command.py` file above `handle_remove_command`. I'll move it there.